### ISSUE TITLE
hmf pipeline for ecdna analysis

### ISSUE THREAD

**Issue by zhaowsong** [hartwigmedical/hmftools#600]
Dear Authors,

I noticed that the HMF pipelines also reported ecDNA events. After analyzing our osteosarcoma PDX WGS data, I found that many samples (90%) exhibited chromothripsis events (ShatterSeek). Additionally, the AA pipeline identified ecDNA events in 50% of the samples. However, the ecDNA events reported by the HMF pipelines were significantly fewer, falling below the 50% threshold.

For instance, in one case, both the AA pipeline and LINX reported ecDNA events. 
<img width="434" alt="image" src="https://github.com/user-attachments/assets/9dc1e801-930e-45dd-a72d-172c72c41cf8">

However, in another case, the AA pipeline identified ecDNA, but LINX did not.
<img width="623" alt="image" src="https://github.com/user-attachments/assets/8d48ac87-7bd6-4060-9288-65bbc7628874">
<img width="400" alt="image" src="https://github.com/user-attachments/assets/dcb555a9-d533-4d7e-bcd3-b2c69537284a">

I would appreciate hearing your thoughts on this.

Best regards,
Zhao

**Comment by zhaowsong** [comment]
Below are the results from the AA pipeline.
<img width="809" alt="image" src="https://github.com/user-attachments/assets/791a77f7-65f6-4802-a5f4-e10955fbe102">

**Comment by p-priestley** [comment]
Hi Zhao,

That is a very complex event with regions oif high copy number on nearly every chromosome.   The picture does not resolve amazingly, so is difficult for me to have a clear view without reviewing the variants.   If I interpret the results from AA it appears to suggest there are 2 ecDNA formed - a 70k  region around MYC and a 3MB region on chr4 apparently from a sigle duplication.  But these regions don't appear to have copy numbers massively higher than surrounding regions, so it does not look compelling to me and the alternative hypothesis is that these duplications are simply part of the wider linear amplification which may have undergone multiple cycles of breakage fusion bridge and shattering.

We did a comparison with AA internally a couple of years ago for ecDNA.  We found very good concordance on Glioblastoma samples but poor concordance on other solid cancer types where AA tended to call many more ecDNA.   We noted we have stricter rules about both the minimum junction copy number for ecDNA and also about excluding BFB.   We don't have a clear truthset so cannot say for sure.

Hope this is helpful

Peter

**Comment by zhaowsong** [comment]
Dear Dr. Peter,

Thank you for your response. I have also reviewed the ecDNA events identified by both LINX and AA. As you mentioned, the copy numbers of these amplicons are significantly higher than those of the surrounding regions and are typically confined to a single chromosome. In a recent preprint about the AA pipeline, these events appear to fall under the unichromosomal category.
<img width="419" alt="image" src="https://github.com/user-attachments/assets/d74d53b7-afaf-452f-a5de-fcb8235663f5">

For the sample mentioned below, both LINX and AA have reported NFIB ecDNA amplification. I have obtained FISH images from the tissue, but they are in interphase.
<img width="504" alt="image" src="https://github.com/user-attachments/assets/e684a53e-76be-4cda-b707-4d8fa75d3a91">

Regarding the sample with MYC amplification that we discussed earlier; I have also acquired interphase FISH images. However, metaphase FISH will still be required for further confirmation.
<img width="522" alt="image" src="https://github.com/user-attachments/assets/102e27c9-1048-428f-9340-376528ca9e6a">

Furthermore, in our osteosarcoma WGS analysis, many amplicons exhibited very complex events involving multiple chromosomal rearrangements. More metaphase FISH will be needed to confirm these analysis results.

Best regards,
Zhao
